rails_app/.rubocop.yml

```diff
@@ -0,0 +1,66 @@
+# The guiding principle of classes is SRP, SRP can't be accurately measured by LoC
+Metrics/ClassLength:
+  Max: 1500
+
+Metrics/MethodLength:
+  Max: 20
+
+Metrics/LineLength:
+  Max: 120
+
+Metrics/AbcSize:
+  Max: 17
+
+Rails:
+  Enabled: true
+Rails/Delegate:
+  Enabled: false
+Rails/HttpPositionalArguments:
+  Enabled: false
+
+# Most readable form.
+Style/AlignHash:
+  EnforcedHashRocketStyle: table
+  EnforcedColonStyle: table
+
+Style/AlignParameters:
```

Comment:
Same as before, didn't find that one in [docs](https://rubocop.readthedocs.io/en/latest/cops_style/)